SPECS/iputils/iputils.spec

```
@@ -1,54 +1,46 @@
-Summary:          Programs for basic networking
-Name:             iputils
-Version:          20180629
-Release:          5%{?dist}
-License:          BSD-3 and GPLv2+ and Rdisc
-URL:              https://github.com/iputils/iputils
-Group:            Applications/Communications
-Vendor:           Microsoft Corporation
-Distribution:     Mariner
-#Source0:         https://github.com/iputils/iputils/archive/s20180629.tar.gz
-Source0:          %{name}-s%{version}.tar.gz
-BuildRequires:    libcap-devel libgcrypt-devel
-Requires:         libcap
-Requires:         libgcrypt
-Obsoletes:        inetutils
+Summary:        Programs for basic networking
+Name:           iputils
+Version:        20211215
+Release:        1%{?dist}
+License:        BSD-3 AND GPLv2+ AND Rdisc
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/Communications
+URL:            https://github.com/iputils/iputils
+Source0:        https://github.com/iputils/iputils/archive/20211215.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  iproute
+BuildRequires:  libcap-devel
+BuildRequires:  libgcrypt-devel
+BuildRequires:  libxslt
+BuildRequires:  meson
+BuildRequires:  ninja-build
+Requires:       libcap
+Requires:       libgcrypt
+Obsoletes:      inetutils
 
 %description
 The Iputils package contains programs for basic networking.
+
 %prep
-%setup -q -n %{name}-s%{version}
+%autosetup -n %{name}-%{version}
 
 %build
-make %{?_smp_mflags} USE_IDN=no USE_GCRYPT=yes
-(
-cd ninfod
-./configure --prefix=%{_prefix}
-make %{?_smp_mflags}
-)
-#make html
-#make man
+%meson
+%meson_build
 
 %install
-rm -fr %{buildroot}
-mkdir -p %{buildroot}%{_sbindir}
-mkdir -p %{buildroot}%{_bindir}
-mkdir -p %{buildroot}/%{_unitdir}
-
-install -c clockdiff %{buildroot}%{_sbindir}/
-install -cp arping %{buildroot}%{_sbindir}/
-install -cp ping %{buildroot}%{_bindir}/
-install -cp rdisc %{buildroot}%{_sbindir}/
-install -cp tracepath %{buildroot}%{_bindir}/
-install -cp traceroute6 %{buildroot}%{_bindir}/
-install -cp ninfod/ninfod %{buildroot}%{_sbindir}/
+%meson_install
```

How about we add a `meson_test` ?
```suggestion
%meson_install


%check
%meson_test
```

---

I am seeing different behavior on my side. Specifically failures on a single test `ping -6 -c1 localhost` . This may be by design since the vm does not have an ipv6 address mapping to hostname in /etc/host. I will add the tests here and leave a comment/